docs/src/man/getting_started.md

```diff
@@ -95,8 +95,8 @@ ncols = size(df, 2)
 We can also look at small subsets of the data in a couple of different ways:
 
 ```julia
-head(df)
-tail(df)
+DataFrames.head(df)
```

Comment:
oh, weird. For some reason it wasn't working for me earlier unless it was fully qualified. I assumed this was because it wasn't exported, but that doesn't actually seem to be the case. Carry on.